.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  jsxBracketSameLine: true,
+  singleQuote: true,
+  trailingComma: 'all',
+  printWidth: 150,
```

Comment:
Hi @hnthangit , i edited to 150 as your mention. Pls re-check and confirm.
Thanks

Reply:
This is optional, feel free